tools/c7n_azure/c7n_azure/policy.py

```diff
@@ -168,6 +170,11 @@ def run(self, event=None, lambda_context=None):
         raise NotImplementedError("subclass responsibility")
 
     def provision(self):
+        if sys.version_info[0] < 3:
+            self.log.error("Cloud Custodian doesn't support Python 2.7 with Azure Functions mode.")
```

Comment:
Not sure why but this sounds funny to me.  I might just say "Python 2.7 is not supported for deploying Azure Functions."